js/pages.js

```diff
@@ -0,0 +1,128 @@
+// Fetch page data from WordPress
```

Comment:
This file is for all pages. It fetches the data for that page. If new data exists from the CMS, we append it to some element with `appendData()` in `main.js`.

Using IDs to target most things.
Data from the CMS is set  up to use the same fields on each page (i.e. `section_1`, `hero_image`) and the pages use the same IDs (i.e. `#section-1`, `#hero-image`).